rpcs3/rpcs3qt/save_manager_dialog.cpp

```diff
@@ -203,6 +276,19 @@ void save_manager_dialog::UpdateList()
 	resize(preferredSize.boundedTo(maxSize));
 }
 
+void save_manager_dialog::UpdateIcons()
+{
+	for (int i = 0; i < m_list->rowCount(); i++)
+	{
+		QTableWidgetItem* item = m_list->item(i, 0);
+		QPixmap data = item->data(Qt::UserRole).value<QPixmap>();
```

Comment:
I was wondering if these won't get too pixelated in hidpi modes

Reply:
I don't have access to hidpi screen so I can't test it, but theoretically this should be fine.

Reply:
I did this in the game list and it was much appreciated